steps/firebase-app-distribution/1.0.0/step.yml

```diff
@@ -0,0 +1,111 @@
+title: Firebase App Distribution
+summary: Distribution pre-release version of your app to your trusted testers with Firebase App Distribution
+description: Firebase App Distribution gives a holistic view of your beta testing program across iOS and Android, providing you with valuable feedback before a new release is in production. You can send pre-release versions of your app using the console or your CI servers, and installing your app is easy for testers.
+website: https://github.com/d-date/bitrise-step-firebase-app-distribution
+source_code_url: https://github.com/d-date/bitrise-step-firebase-app-distribution
+support_url: https://github.com/d-date/bitrise-step-firebase-app-distribution/issues
+published_at: 2019-09-28T22:08:04.945934+02:00
+source:
+  git: https://github.com/d-date/bitrise-step-firebase-app-distribution.git
+  commit: 2e3120f6c72e583c63c67a18b51eaa67034d7e38
+host_os_tags:
+- osx-10.10
+- ubuntu-16.04
+project_type_tags:
+- ios
+- android
+- xamarin
+- react-native
+- cordova
+- ionic
+type_tags:
+- deploy
+toolkit:
+  bash:
+    entry_file: step.sh
+deps:
+  brew:
+  - name: git
+  - name: wget
+  apt_get:
+  - name: git
+  - name: wget
+is_requires_admin_user: true
+is_always_run: false
+is_skippable: false
+run_if: ""
+inputs:
+- opts:
+    description: |
+      Required. Your firebase token for CLI. https://firebase.google.com/docs/cli#cli-ci-systems
+    is_required: true
+    title: Firebase Token
+  token: $FIREBASE_TOKEN
+- ipa_path: $BITRISE_IPA_PATH
+  opts:
+    is_required: true
```

Comment:
What about APKs? `project_type_tags` list also Android.

Reply:
APK is also available with same interface. Please let me know it should be.

Reply:
Then input should be called `APP_FILE_PATH` or something like that. And description should contain info that it may point to either APK or IPA.